Fix #1368: Modify datatype of pa_activation.extras (#1371)
wultra/powerauth-server: docs/PowerAuth-Server-1.7.0.md

- `external_id` - external identifier of the activation
- `protocol` - protocol enumeration: `powerauth` or `fido2`

> The data type for column `extras` in table `pa_activation` was changed to `VARCHAR` / `VARCHAR2` to support larger data.

Reviewer: ```suggestion
The size of column `extras` in table `pa_activation` was increased to `4000` from `255` to support larger data.
```
